docs/Contributing.asciidoc

```diff
@@ -163,7 +163,7 @@ there are some steps that have to be followed so that new database instances
 and upgrades include those changes.
 
 .  First, you need to increase the database version number in the $VERSION
-   variable at the first lines of the +lib/OpenQA/Schema/Schema.pm+ file.
+   variable at the first lines of the +lib/OpenQA/Schema.pm+ file.
```

Comment:
Please leave out 'at the first lines'. And mark `$VERSION` as variable by backquoting it.